app/models/post.js

```diff
@@ -19,4 +19,6 @@ export default DS.Model.extend({
   project: DS.belongsTo('project', { async: true }),
   user: DS.belongsTo('user', { async: true }),
   comments: DS.hasMany('comment', { async: true }),
+
+  postUserMentions: DS.hasMany('post-user-mention', { asnyc: false })
```

Comment:
`asnyc` should be `async`. Also, should this be `true` and not `false`? I know that's not how it is in API now, but should it be?